python/lsst/sconsUtils/scripts.py

```diff
@@ -467,6 +491,8 @@ def tests(pyList=None, ccList=None, swigNameList=None, swigSrc=None,
             ignoreList = []
 
         def s(l):
+            if l is None:
+                return ['None']
             return [str(i) for i in l]
```

Comment:
I know this is prior formatting but a newline would be nice here to visually separate the function block.  I'm surprised that flake8 doesn't catch nested functions.